osmaxx/conversion/converters/converter_gis/bootstrap/bootstrap.py

```diff
@@ -3,17 +3,16 @@
 
 from memoize import mproperty
 
+from osmaxx.conversion.converters.converter_gis.detail_levels import DETAIL_LEVEL_ALL, DETAIL_LEVEL_TABLES
```

Comment:
Consistency: I would prefer, having the same import style everywhere, ie. `from osmaxx.conversion.converters.converter_gis.detail_levels import DETAIL_LEVEL_ALL, DETAIL_LEVEL_TABLES` 
*OR*
 `from osmaxx.conversion.converters.converter_gis import detail_levels`